integration-box/yahoo-dmp/scripts/put_userlist.py

```diff
@@ -40,4 +40,4 @@ def upload(sqlfile, database, presigned_url):
     logger.error(res.text)
     sys.exit(os.EX_DATAERR)
   else:
-    print(f"Succeeded calling Yahoo API with http status code {res.status_code}")
+    logger.info(f"Succeeded calling Yahoo API with http status code {res.status_code}")
```

Comment:
[nits] [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation) recommends 4 space indentation. Might be better to apply `black` as a formatter.

Reply:
@chezou Thanks! I've just install `black` and applied it!